syntax = "proto3";

message Input {
    string model = 1;
    string img_path = 2;
    string confidence = 3;
}

message Output {
    string delta_time = 1;
    string boxes = 2;
    string class_ids = 3;
    string confidences = 4;
    string img_base64 = 5;
}

service Detect {
    rpc detect(Input) returns (Output) {}
}
